res/layout/contact_item.xml

```diff
@@ -18,14 +18,14 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginLeft="7dip"
-    android:paddingLeft="4dip"
+    android:layout_marginStart="7dip"
```

Comment:
Using `*Start` and `*End` instead of `*Left` and `*Right` will make the app initially supporting RTL, which is good 😄 , however, the content view of where the days are shown will be the opposite side. because [SimpleWeekView.java](https://github.com/Etar-Group/Etar-Calendar/blob/master/src/com/android/calendar/month/SimpleWeekView.java) doesn't support RTL. This will make a confusion to the user. That's why I recommend not do it until fixing (Support RTL #220).